test/extended/operators/olm.go

```diff
@@ -226,4 +226,28 @@ var _ = g.Describe("[Feature:Platform] an end user use OLM", func() {
 		}
 
 	})
+
+	// OCP-24818 - Checking OLM descriptors
+	// author: tbuskey@redhat.com
+	g.It("Checking OLM descriptors", func() {
```

Comment:
Is this being done for all CRDs shipped with OpenShift?

Reply:
@jianzhangbjz Do you have direction here?

Reply:
This test seems extremely similar to those featured in https://github.com/openshift/origin/pull/22696 as well.

Reply:
We were directed to move QE OLM tests out of the origin repo to this one.

Reply:
Okay - that's good to know - thanks!

Reply:
Yes, we're migrating the extended tests from origin to this repo. Details in: https://github.com/openshift/enhancements/pull/183